sys/parsoid.js

```diff
@@ -236,72 +394,47 @@ class ParsoidService {
             const pageBundleUri = new URI([rp.domain, 'sys', 'parsoid', 'pagebundle',
```

Comment:
Perhaps this could be simplified so that the pagebundle method is called directly instead of going through the router?

Reply:
There's a lot of simplification opportunities here but I don't want to touch anything not strictly required to be touched at this point.